<ui version="4.0" >
 <class>AddIdentityBase</class>
 <widget class="QWidget" name="AddIdentityBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Identity Assistant</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="KTitleWidget" name="title" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="text" >
      <string>Add New Identity</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="identityNameLabel" >
       <property name="text" >
        <string>New identity:</string>
       </property>
       <property name="buddy" >
        <cstring>identityName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="identityName" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="newOption" >
     <property name="text" >
      <string>Create an empty identity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="duplicateOption" >
     <property name="text" >
      <string>Duplicate an existent identity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="identityList" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Identity to Duplicate</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
